src/TaskoPhobia.Core/DomainServices/InvitationService.cs

```diff
@@ -2,19 +2,22 @@
 using TaskoPhobia.Core.Exceptions;
 using TaskoPhobia.Core.Policies;
 using TaskoPhobia.Core.ValueObjects;
+using TaskoPhobia.Shared.Abstractions.Time;
 
 namespace TaskoPhobia.Core.DomainServices;
 
 public sealed class InvitationService : IInvitationService
 {
+    private readonly IClock _clock;
     private readonly IEnumerable<IInvitationPolicy> _policies;
 
-    public InvitationService(IEnumerable<IInvitationPolicy> policies)
+    public InvitationService(IEnumerable<IInvitationPolicy> policies, IClock clock)
     {
         _policies = policies;
+        _clock = clock;
     }
 
-    public void CreateInvitationToProject(Project project, UserId senderId, UserId receiverId, Invitation invitation)
+    public void CreateInvitationToProject(Project project, UserId senderId, Invitation invitation)
     {
         var policy = _policies.FirstOrDefault(x => x.CanBeApplied());
```

Comment:
Można to sobie połączyć w jednym zapytaniu żeby znaleźć politykę która pozwala na CanBeAppllied i CanCreate